### ISSUE TITLE
When searching for a CVE, suggest packages. When searching for a package, suggest CVEs

### ISSUE THREAD

**Issue by fricklerhandwerk** [Nix-Security-WG/nix-security-tracker#177]

**Comment by erictapen** [comment]
Thinking about this more, I'm not even sure we need a suggestion for CVEs, as my current understanding of the triage workflow is, that the security team is only busy with working through all the new CVEs that are not marked yet.

So in the triage view, the primary concern would be to see a list of uncategorised CVEs that are assigned to a package one by one.

**Comment by fricklerhandwerk** [comment]
Indeed, for package maintainers we'd need the inverse view. Each package maintainer "knows" their packages, and they may want automatic suggestions for CVEs. Both security team and package maintainers should be able to search back and forth though.
